backend/blueprint/api/admin/users.py

```diff
@@ -0,0 +1,254 @@
+from flask import Blueprint, jsonify, request
+from api.utils.response import Response
+from firebase_admin import auth
+
+
+router = Blueprint('users', __name__)
+
+
+@router.route('/createuser', methods=['POST'])
+def create_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+
+    try:
+        user = auth.create_user(
+            display_name=data['name'],
+            email=data['email'],
+            email_verified=data['email_verified'],
+            photo_url=data['photo_url'],
+            password=data['password'],
+            disabled=data['disabled']
+        )
+        return jsonify({
```

Comment:
We have a response class, please use that.

Reply:
@djmgit if wanna show only `Sucessfully created new user` then we can use Response class, if we want to update the user with the `uid` after creating the user then we can't use Response class, we have to modify the response class